### PR TITLE
Changing pull_request changed to pull_request_target to fix build failure from fork

### PR DESCRIPTION
**Pull request by athottungal**

### CODE REVIEW

**Review by tussharrane — APPROVED** [review]
For Testing purpose!

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -35,5 +37,16 @@ jobs:
...
       - uses: actions/checkout@v2
+        with: 
+          persist-credentials: false
       - name: Test Coverage
         uses: ArtiomTr/jest-coverage-report-action@v2
+        id: coverage
+        with:
+            # tell to the action to not attach comment.
+            output: report-markdown
+            annotations: none
```
Comment (mmoss) [inline .github/workflows/checks.yml:48]: Let's try this to see if we can get this working the same across forks and the root repo:

```js
            output: ${{ github.event.pull_request.head.repo.full_name == github.event.repository.name && 'comment' || 'report-markdown' }}
            annotations: ${{ github.event.pull_request.head.repo.full_name == github.event.repository.name && 'coverage' || 'none' }}

```

Comment (athottungal) [reply .github/workflows/checks.yml]: done

**Review by mmoss — APPROVED** [review]

**Review by dannyscot — APPROVED** [review]
LGTM 👍 

why the "Safe" label? I think I missed that

**Pull request merged**